packages/components/src/components/single-select/shadow.tsx

```diff
@@ -0,0 +1,658 @@
+	}
+
+	@Watch('_placeholder')
+	public validatePlaceholder(value?: string): void {
+		this.controller.validatePlaceholder(value);
+	}
+
+	@Watch('_accessKey')
+	public validateAccessKey(value?: string): void {
+		this.controller.validateAccessKey(value);
+	}
+
+	@Watch('_alert')
+	public validateAlert(value?: boolean): void {
+		this.controller.validateAlert(value);
+	}
+
+	@Watch('_disabled')
+	public validateDisabled(value?: boolean): void {
+		this.controller.validateDisabled(value);
+	}
+
+	@Watch('_hideError')
+	public validateHideError(value?: HideErrorPropType): void {
+		this.controller.validateHideError(value);
+	}
+
+	@Watch('_hideLabel')
+	public validateHideLabel(value?: boolean): void {
+		this.controller.validateHideLabel(value);
+	}
+
+	@Watch('_hint')
+	public validateHint(value?: string): void {
+		this.controller.validateHint(value);
+	}
+
+	@Watch('_icons')
+	public validateIcons(value?: Stringified<KoliBriHorizontalIcons>): void {
+		this.controller.validateIcons(value);
+	}
+
+	@Watch('_id')
+	public validateId(value?: string): void {
+		this.controller.validateId(value);
+	}
+
+	@Watch('_label')
+	public validateLabel(value?: LabelWithExpertSlotPropType): void {
+		this.controller.validateLabel(value);
+	}
+
+	@Watch('_msg')
+	public validateMsg(value?: MsgPropType): void {
+		this.controller.validateMsg(value);
+	}
+
+	@Watch('_name')
+	public validateName(value?: string): void {
+		this.controller.validateName(value);
+	}
+
+	@Watch('_on')
+	public validateOn(value?: InputTypeOnDefault): void {
+		this.controller.validateOn(value);
+	}
+
+	@Watch('_options')
+	public validateOptions(value?: OptionsPropType): void {
+		this.controller.validateOptions(value);
+		this._filteredOptions = value;
+	}
+
+	@Watch('_required')
+	public validateRequired(value?: boolean): void {
+		this.controller.validateRequired(value);
+	}
+
+	@Watch('_syncValueBySelector')
+	public validateSyncValueBySelector(value?: SyncValueBySelectorPropType): void {
+		this.controller.validateSyncValueBySelector(value);
+	}
+
+	@Watch('_tabIndex')
+	public validateTabIndex(value?: number): void {
+		this.controller.validateTabIndex(value);
+	}
+
+	@Watch('_touched')
+	public validateTouched(value?: boolean): void {
+		this.controller.validateTouched(value);
+	}
+
+	@Watch('_value')
+	public validateValue(value?: string): void {
+		this.controller.validateValue(value);
+		this.oldValue = value;
+	}
+
+	public componentWillLoad(): void {
+		this.refOptions = [];
+		this._alert = this._alert === true;
+		this._touched = this._touched === true;
+		this.controller.componentWillLoad();
+		this.oldValue = this._value;
+		this._filteredOptions = this._options;
+
+		if (Array.isArray(this._options)) {
+			const matchedOption = this._options.find((option) => (option as Option<string>).value === this._value);
+			this._inputValue = matchedOption ? (matchedOption.label as string) : '';
+		}
+		this.host?.addEventListener('focusout', () => {
```

Comment:
In [dem Beispiel](https://codepen.io/sdvg/pen/VwJZXgd), das ich für die Handhabung von "Focus Out" gebaut hatte, wird auch das Window Blur berücksichtigt. Das vermisse ich hier leider noch.